README.md

```diff
@@ -117,7 +117,7 @@ Then to install the C library:
 git clone https://github.com/openvenues/libpostal
 cd libpostal
 ./bootstrap.sh
-./configure --datadir=[...some dir with a few GB of space...]
+./configure --datadir=[...some dir with a few GB of space where a "libpostal" directory exists or can be created/modified...]
```

Comment:
Same change should be made to line 165 (under the windows build section) to keep consistent.

Reply:
Thank you ! :)